test/src/main/java/org/apache/accumulo/test/functional/BulkSplitOptimizationIT.java

```diff
@@ -105,6 +107,8 @@ public void testBulkSplitOptimization() throws Exception {
       // initiate splits
       c.tableOperations().setProperty(tableName, Property.TABLE_SPLIT_THRESHOLD.getKey(), "100K");
 
+      c.tableOperations().setTabletHostingGoal(tableName, new Range(), TabletHostingGoal.ALWAYS);
```

Comment:
I added that so I could run the test and see it pass.  It should eventually work w/o that. I'll remove it now because we will probably forget about it.

Reply:
removed in [27fc328](https://github.com/apache/accumulo/pull/3350/commits/27fc328d10979710023da6207f9eba92250dca3b)